src/coreclr/vm/jitinterface.cpp

```diff
@@ -674,27 +675,35 @@ const char16_t* CEEInfo::getStringLiteral (
 
     Module* module = GetModule(moduleHnd);
 
-    const char16_t* result = nullptr;
+    _ASSERTE(bufferSize >= 0);
+
+    int result = -1;
 
     JIT_TO_EE_TRANSITION();
 
+    ULONG dwCharCount;
```

Comment:
Nit: The old version where `dwCharCount` and `pString` were right next to the call to `GetUserString` looked better.